examples/minio/listobjectsV2WithMetadata.go

```diff
@@ -37,7 +37,7 @@ func main() {
 
 	// New returns an Amazon S3 compatible client object. API compatibility (v2 or v4) is automatically
 	// determined based on the Endpoint value.
-	s3Client, err := minio.New("s3.amazonaws.com", &minio.Options{
+	minioClient, err := minio.New("play.min.io", &minio.Options{
```

Comment:
```suggestion
	s3Client, err := minio.New("play.min.io", &minio.Options{
```

Maybe provide full credentials.